commons/src/testFixtures/java/one/tomorrow/transactionaloutbox/commons/ProxiedPostgreSQLContainer.java

```diff
@@ -13,19 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package one.tomorrow.transactionaloutbox.reactive;
+package one.tomorrow.transactionaloutbox.commons;
 
+import eu.rekawek.toxiproxy.Proxy;
+import org.jetbrains.annotations.NotNull;
 import org.springframework.test.context.DynamicPropertyRegistry;
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.PostgreSQLContainer;
 import org.testcontainers.containers.ToxiproxyContainer;
 import org.testcontainers.utility.DockerImageName;
 
-public class ProxiedPostgreSQLContainer extends PostgreSQLContainer<ProxiedPostgreSQLContainer> {
+import static one.tomorrow.transactionaloutbox.commons.ProxiedContainerPorts.findPort;
+import static one.tomorrow.transactionaloutbox.commons.ProxiedContainerSupport.createProxy;
+
+public class ProxiedPostgreSQLContainer extends PostgreSQLContainer<ProxiedPostgreSQLContainer> implements ProxiedContainerSupport {
 
     private static ProxiedPostgreSQLContainer postgres;
     private static ToxiproxyContainer toxiproxy;
-    public static ToxiproxyContainer.ContainerProxy postgresProxy;
+    public static Proxy postgresProxy;
```

Comment:
Why do you not use the `ContainerProxy` anymore? As I can see, is the `ContainerProxy` part of the library we're using, but the `Proxy` is coming from somewhere else.

Reply:
Well, because it's deprecated :-)

Reply:
See e.g.
* https://github.com/Shopify/toxiproxy/issues/470
* https://java.testcontainers.org/modules/toxiproxy/

Reply:
I'm already marking this as resolved, if there's anything else we can still adjust it.